include/grpc/impl/codegen/grpc_types.h

```diff
@@ -285,9 +285,11 @@ typedef struct {
 #define GRPC_ARG_SOCKET_MUTATOR "grpc.socket_mutator"
 /** The grpc_socket_factory instance to create and bind sockets. A pointer. */
 #define GRPC_ARG_SOCKET_FACTORY "grpc.socket_factory"
-/** The maximum number of trace events to keep in the tracer for each channel or
- * subchannel. The default is 10. If set to 0, channel tracing is disabled. */
-#define GRPC_ARG_MAX_CHANNEL_TRACE_EVENTS_PER_NODE \
+/** The maximum ammount of memory that the trace event can use per channel
```

Comment:
The wording "memory that the trace event can use" makes this seem like this limit is per-event, not per-node.  Suggest changing this to "memory used by trace events".

Reply:
Done